pkg/ui/handlers.go

```diff
@@ -32,6 +56,10 @@ func (a *API) uiFiles(w http.ResponseWriter, r *http.Request) {
 func NewAPI(fileSystem fs.FS, logger logging.LoggerInterface) *API {
 	a := new(API)
 
+	a.fileSystem = fileSystem
+
+	a.servePattern, _ = regexp.Compile(`(svg|txt|html)$`)
```

Comment:
I am a little concerned that if someone wants to add a new file to the UI they will also have to change this.